pkg/processor/k8sprocessor/kube/client.go

```diff
@@ -117,6 +119,51 @@ func New(
 	return c, err
 }
 
+// OwnerFeedbackLoop starts a loop that will receive notifications about updates
+// to endpoints objects which are then used to update corresponding pods with
+// ownership info.
+// For info it only updates the services attribute. See
+// Test_PodIsAnnotatedWithOwnershipInfoImmediatelyAfterOwnersAreCreated test in
+// client_test.go for details.
+func (c *WatchClient) OwnerFeedbackLoop(stopCh <-chan struct{}) {
+	ch := c.op.GetPodUpdateChan()
+	for {
+		select {
+		case <-stopCh:
+			return
+
+		case pod, ok := <-ch:
+			if !ok {
+				return
+			}
+
+			p, ok := c.GetPod(generatePodID(pod))
+			if !ok {
+				continue
+			}
+
+			if c.Rules.OwnerLookupEnabled {
```

Comment:
Is this check necessary? We're only creating the owner cache and starting this loop if this is enabled.